colour-changing-grid/script.js

```diff
@@ -1 +1,16 @@
-// colour changing grid JS here
\ No newline at end of file
+// colour changing grid JS here
+function randomNumber (range) {
+	return Math.floor(Math.random()*range);
+};
+function mOver(obj) {
```

Comment:
Don't be afraid to be verbose and exact when naming variables and functions. When just glancing at the JS both mOut and mOver don't have immediate recognition.

Make things as simple as possible, something similar to handleMouseOver and handleMouseOut would be decent examples to replace them.